### ISSUE TITLE
problems about keeping backing vocals while removing vocals 

### ISSUE THREAD

**Issue by NoahMarva** [Anjok07/ultimatevocalremovergui#231]
Hello! This is my first time here and I'm not sure if my question is appropriate. I've found a common problem with many spleeter based services that remove the backing vocals while removing the vocals, I've only found one service provider online that can do this while removing the vocals and keeping the backing vocals. Can this be done?

**Comment by Anjok07** [comment]
Yes, UVR has Karaoke models available that remove main vocals only.

**Comment by NoahMarva** [comment]
Thanks, that's just great to know! You guys are amazing.XD


| |
Xinyi
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/26/2022 ***@***.***> wrote：

Yes, UVR has Karaoke models available that remove main vocals only.

**Comment by NoahMarva** [comment]
Hello! Sorry to follow up on this, but do you have a web version? My laptop is showing insufficient GPU memory.


| |
Xinyi
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/26/2022 ***@***.***> wrote：
Thanks, that's just great to know! You guys are amazing.XD


| |
Xinyi
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/26/2022 ***@***.***> wrote：

Yes, UVR has Karaoke models available that remove main vocals only.
